### PR TITLE
Jenkins Node Reboot

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: tests/e2e/workloads/jenkins/test_jenkins_node_reboot.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        # Init number of builds per project
+        jenkins.number_builds_per_project = num_of_builds
+
+        # Start Builds
+        jenkins.start_build()
+
+        if len(nodes_reboot) > 0:
+            # Restart Node
+            nodes.restart_nodes(get_node_objs(nodes_reboot))
+
```
Comment (hnallurv) [inline tests/e2e/workloads/jenkins/test_jenkins_node_reboot.py:93]: where do you check whether the rebooted node came back up or not?

Comment (OdedViner) [reply tests/e2e/workloads/jenkins/test_jenkins_node_reboot.py:93]: "restart_nodes" function checks that node come back up.
image

Comment (OdedViner) [reply tests/e2e/workloads/jenkins/test_jenkins_node_reboot.py:96]: I added "restart_nodes_by_stop_and_start_teardown" function on tear-down step.
This function  make sure all Nodes are up by the end of the test

**Comment by vasukulkarni** [comment]
Added "needs testing" label, we have the test failure issue here in Jenkins that needs to be fixed.

**Pull request merged**